api/webhooks/webhooks.py

```diff
@@ -101,9 +118,14 @@ def _call_webhook(
     return requests.post(str(webhook.url), data=json_data, headers=headers, timeout=10)
 
 
+@register_task_handler()
 def _call_webhook_email_on_error(
-    webhook: WebhookModels, data: typing.Mapping, webhook_type: WebhookType
+    webhook_id: int, data: typing.Mapping, webhook_type: WebhookType
```

Comment:
I think `webhook_type: WebhookType` might be wrong here? It should be `webhook_type: str` I think?

Reply:
There aren't _yet_ but it's in the plans so we're trying to ensure we're chipping away at the typing. You're right that it would be good to have something that validates the diff only for now.